icons/fontra-icons.ufo/glyphs/skew.glif

```diff
@@ -0,0 +1,183 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<glyph name="skew" format="2">
+  <advance width="938"/>
```

Comment:
The width should be 1000. Please also check the vertical alignment compared to the other icons: this one sits a bit lower than the rest.

Reply:
Interesting, I did set the width to 1000 at the beginning. I am wondering what caused the change? Anyway, thanks for finding it, let me fix it.

Reply:
Fix it.